src/elevateprivilegeshelper.h

```diff
@@ -0,0 +1,76 @@
+/*
+    SPDX-FileCopyrightText: Zeno Endemann <zeno.endemann@kdab.com>
+    SPDX-FileCopyrightText: 2016-2023 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+
+    SPDX-License-Identifier: GPL-2.0-or-later
+*/
+
+#pragma once
+
+#include <QObject>
+#include <QString>
+#include <QStringList>
+
+#include <unistd.h>
+
+class QSocketNotifier;
+
+class InitiallyStoppedProcess
```

Comment:
could you do me a favor and add basic api documentation to this class and its methods? I guess run can only be executed after reset?